eng/common/scripts/TypeSpec-Project-Process.ps1

```diff
@@ -71,6 +60,20 @@ function CreateUpdate-TspLocation([System.Object]$tspConfig, [string]$TypeSpecPr
   return $packageDir
 }
 
+function Get-ServiceDir([System.Object]$tspConfig, [string]$repoRoot) {
+  $serviceDir = ""
+  if ($tspConfig["parameters"] -and $tspConfig["parameters"]["service-dir"]) {
+    $serviceDir = $tspConfig["parameters"]["service-dir"]["default"];
+  }
+  else {
+    Write-Error "Missing service-dir in parameters section of tspconfig.yaml."
```

Comment:
We should probably link to some docs so folks know how to fix this.

Reply:
I added the sample project tspconfig.yaml for latest schema.